src/adminEntity.ts

```diff
@@ -88,6 +89,18 @@ abstract class AdminEntity {
       }
     })
   }
-}
 
+  validateFormConfig() {
+    const count = countBy(this.fields)
+
+    Object.keys(count).forEach(key => {
+      if (count[key] > 1) throw new InvalidDisplayFieldsException(`Property ${key} is duplicated`)
```

Comment:
It's not about the `displayFields` attribute, this exception doesn't make sense to throw here. We could create an `InvalidAdminEntityFormConfig` exception